src/ansys/fluent/core/docker/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  fluent:
+    image: ${IMAGE_NAME}
+    environment:
+      - IMAGE_NAME=${IMAGE_NAME}
+      - ANSYSLMD_LICENSE_FILE=${LICENSE_FILE}
+      - REMOTING_PORTS=${REMOTE_PORTS}
+      - FLUENT_NO_AUTOMATIC_TRANSCRIPT=${NO_TRANSCRIPT}
+      - COMMAND=${COMMAND}
+      - PORT_1=${PORT_1}
+      - PORT_2=${PORT_2}
+      - MOUNT_SOURCE=${MOUNT_SOURCE}
+      - MOUNT_TARGET=${MOUNT_TARGET}
+    command: ${COMMAND}
+    ports:
+      - ${PORT_1}:${PORT_1}
+      - ${PORT_2}:${PORT_2}
```

Comment:
This is odd, but I see there is an open issue in docker compose to support settings array of values via env var (https://github.com/docker/compose/issues/4249).

Reply:
https://stackoverflow.com/a/72498747 - this might be the current workaround to set a variable number of ports programmatically.

Reply:
There is no other way to map multiple ports to service through docker compose, we have to specify them explicitly.

Reply:
> https://stackoverflow.com/a/72498747 - this might be the current workaround to set a variable number of ports programmatically.

@mkundu1 It writes ports to docker-compose.yaml file directly which is not suitable for multiple instances using same docker-compose.yaml as we need the same compose file for both start and stop the container.

Reply:
Ok. Let me try this.

Reply:
@mkundu1 Done. Thank you.